### PR TITLE
[DEV-2194] change limit request mail (#1088)

### PR DESCRIPTION
**Pull request by Yannick1712**
#### Pre-Release

- [x] Check migrations
  - No database related infos (sqldb-xxx)
  - Impact on GS (new/removed columns)
- [x] Check for linter errors (in PR)
- [x] Test basic user operations
  - Login/logout
  - Registration
  - Update user data
  - Create/delete payment routes

#### Post-Release

- Test basic user operations
- Monitor application insights log

### CODE REVIEW

**Comment by Yannick1712** [comment]
Update env when merging on prd (Support Banner)

File: src/shared/i18n/es/mail.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
         "thanks": "Muchas gracias por tu confianza",
         "happy_trading": "¡Que disfrutes al comerciar!",
         "welcome": "Hola {name},",
-        "personal_questions": "Please do not hesitate to contact me if you have any questions.",
+        "team_questions": "If you have any questions, please do not hesitate to contact us.",
         "personal_closing": "Kind regards,<br>{closingName}",
         "button": "o<br>[url:Haga clic aquí]"
```
Comment (bernd2022) [inline src/shared/i18n/es/mail.json:11]: "ES" Translation missing

File: src/shared/i18n/fr/mail.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
         "thanks": "Merci beaucoup pour votre confiance",
         "happy_trading": "Bon trading",
         "welcome": "Bonjour {name},",
-        "personal_questions": "Please do not hesitate to contact me if you have any questions.",
+        "team_questions": "If you have any questions, please do not hesitate to contact us.",
         "personal_closing": "Kind regards,<br>{closingName}",
```
Comment (bernd2022) [inline src/shared/i18n/fr/mail.json:10]: "FR" Translation missing

File: src/shared/i18n/it/mail.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
         "thanks": "Grazie mille per la tua fiducia",
         "happy_trading": "Buon trading",
         "welcome": "Ciao {name},",
-        "personal_questions": "Please do not hesitate to contact me if you have any questions.",
+        "team_questions": "If you have any questions, please do not hesitate to contact us.",
         "personal_closing": "Kind regards,<br>{closingName}",
```
Comment (bernd2022) [inline src/shared/i18n/it/mail.json:10]: "IT" Translation missing

File: src/shared/i18n/pt/mail.json [hunk 4]
```diff
@@ -6,7 +6,7 @@
         "thanks": "Thank you very much for your trust",
         "happy_trading": "Happy Trading",
         "welcome": "Hi {name},",
-        "personal_questions": "Please do not hesitate to contact me if you have any questions.",
+        "team_questions": "If you have any questions, please do not hesitate to contact us.",
         "personal_closing": "Kind regards,<br>{closingName}",
```
Comment (bernd2022) [inline src/shared/i18n/pt/mail.json:10]: "PT" Translation missing

File: src/subdomains/supporting/pricing/services/pricing.service.new.ts
Comment (bernd2022) [inline src/subdomains/supporting/pricing/services/pricing.service.new.ts:1]: Shall the filename really be "...new..."?

**Review by bernd2022 — APPROVED** [review]

File: src/subdomains/supporting/pricing/services/pricing.service.new.ts
Comment (davidleomay) [reply src/subdomains/supporting/pricing/services/pricing.service.new.ts:1]: This is only "experimental" 😁  will be renamed with the next PR.

**Review by davidleomay — APPROVED** [review]

**Review by davidleomay — APPROVED** [review]

**Pull request merged**